### ISSUE TITLE
The responsive behavior of ActionSheet need to follow Fiori guide  on in smartphones 

### ISSUE THREAD

**Issue by sleric1024** [SAP/ui5-webcomponents-react#1701]
**Is your feature request related to a problem? Please describe.**
Hi UI5 team colleagues, I am from Successfactors.
We are delivering the features that need to using the ActionSheet component, while the responsive behavior of ActionSheet doesn't follow SAP Fiori guide  on in smartphones.
Demo: https://5qlmu.csb.app/

You may open the demo and using the device simulator of Chrome.
![Screen Shot 2021-06-09 at 9 34 08 AM](https://user-images.githubusercontent.com/5724592/121282776-a3cf6280-c90c-11eb-845e-b6aa26890f2c.png)

**Describe the solution you'd like**
According to Fiori guide line. https://ux.wdf.sap.corp/fiori-design-web/action-sheet/#responsiveness

The expected  display should be like this.
![Screen Shot 2021-06-09 at 9 36 27 AM](https://user-images.githubusercontent.com/5724592/121282973-f577ed00-c90c-11eb-8099-db0912e5b336.png)

**Describe alternatives you've considered**

**Additional context**
The related version we are using.
https://codesandbox.io/s/kind-leavitt-h4jrn?file=/package.json:155-357
```js
    "@ui5/webcomponents": "~0.31.8",
    "@ui5/webcomponents-fiori": "~0.31.8",
    "@ui5/webcomponents-react": "~0.131.6",
    "react": "17.0.1",
    "react-dom": "17.0.1",
    "react-scripts": "4.0.3"
```

**Comment by Lukas742** [comment]
Hi @sleric1024 

this feature is already available in our main version. I did down-port it and it's now also available with `0.131.8`. 

Please always check first if features are already available in our main version. If so, you can also just ping us in the internal Slack channel, so we can down port the changes right away.

**Comment by sleric1024** [comment]
@Lukas742 , got that. Thanks.
